config/webhook/kustomization.yaml

```diff
@@ -4,3 +4,18 @@ resources:
 
 configurations:
 - kustomizeconfig.yaml
+
+patches:
```

Comment:
Could you move this logic as part of the already present namespace_selector_patch ?
https://github.com/openshift/instaslice-operator/blob/main/config/default/namespace_selector_patch.yaml

Reply:
Also, do you think we should update the existing `matchExpressions`?